src/components/FriendPage/FriendPage.js

```diff
@@ -24,22 +26,31 @@ const FriendPage = ({ id }) => {
     )
   }
 
-  return (
-    <main className='friend-gift-page'>
-      <header>
-        <img src={getIcon(id)}></img>
-        <h1>{friend.name}</h1>
-        <h2>{friend.birthday}</h2>
-        <p>{friend.memo}</p>
-      </header>
-      <section className='add-a-gift'>
-        <i className='fas fa-plus fa-3x' onClick={() => setForm(true)}></i>
-        <h3>Add a gift idea</h3>
-        {showForm && displayForm()}
-      </section>
-
-    </main>
-  )
+  if (friend) {
+    return (
+      <main className='friend-gift-page'>
+        <header>
+          <img src={getIcon(id)}></img>
+          <h1>{friend.name}</h1>
+          <h2>{friend.birthday}</h2>
```

Comment:
Later we can have the birthday be displayed the same way as the UpcomingBirthdays component once I get that fixed to say July 7th.